sdk/nodejs/provisioning/bin.ts

```diff
@@ -32,7 +33,9 @@ export class Bin implements EngineConn {
   private binPath?: string
   private cliVersion?: string
 
-  private readonly cacheDir = envPaths("dagger", { suffix: "" }).cache
+  private readonly cacheDir = `${
+    process.env.XDG_CACHE_HOME?.trim() || envPaths("", { suffix: "" }).cache
+  }/dagger`
```

Comment:
Does the forward slash work ok in widows? Maybe better to use `path.join()`?

Reply:
I've just pushed the fix to be sure !